pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -123,6 +230,8 @@ func (e *DockercfgController) Run(workers int, stopCh <-chan struct{}) {
 		go wait.Until(e.worker, time.Second, stopCh)
 	}
 
+	go e.secretController.Run(stopCh)
```

Comment:
need to start both controllers and wait until they have synced before starting the workers